.gitlab-ci.yml

```diff
@@ -14,7 +14,22 @@ include:
     file:
       - templates/global/full.yaml
       - templates/python/full.yaml
+      - templates/common/python.yaml
 
+.python_versions:
+  parallel:
+    matrix:
+      - BUILD_PY_VERSION: [python3.7]
```

Comment:
Shouldn't' there be multiple python versions specified here? One for each supported version?

Reply:
added 3.8 and 3.9